parl/remote/worker.py

```diff
@@ -170,10 +171,13 @@ def _fill_job_buffer(self):
         """An endless loop that adds initialized job into the job buffer"""
         while self.worker_is_alive:
             if self.job_buffer.full() is False:
-                initialized_jobs = self._init_jobs(job_num=self.cpu_num)
+                initialized_jobs = self._init_jobs(
```

Comment:
what if `self.cpu_num` is equal to `self.job_buffer.qisze()` here?

Reply:
It will raise assertion error, have fixed it by adding job_num condition.